Tower Defense Unity Project/Assets/Scripts/WaveSpawner.cs

```diff
@@ -19,9 +19,14 @@ public class WaveSpawner : MonoBehaviour {
 
 	private int waveIndex = 0;
 
-	void Update ()
+    void Start()
+    {
+        EnemiesAlive = 0;
+    }
+
+    void Update ()
```

Comment:
also adding access modifiers to functions would be good. Like `private void Update()... `